Formula/g/goldminer.rb

```diff
@@ -0,0 +1,38 @@
+class Goldminer < Formula
+  desc "A terminal-based Gold Miner game"
+  homepage "https://github.com/kongtaoxing/goldminer"
+  url "https://github.com/kongtaoxing/goldminer/releases/download/v0.0.1/goldminer-mac.tar.gz"
+  sha256 "bdb709f06a6ca11ab2e57335e44bbb98c87aca68eb6549add1ca70618e5b7f31"
+  license "MIT"
+
+  def install
+    bin.install "goldminer"
```

Comment:
We don't allow to install binary in homebrew/core. The formula should build by own code.